code/controllers/DMSCheckoutController.php

```diff
@@ -1,18 +1,21 @@
 <?php
 
-class DMSDocumentCartCheckoutPage_Controller extends Page_Controller
+class DMSCheckoutController extends ContentController
 {
     private static $allowed_actions = array(
         'DMSDocumentRequestForm',
-        'complete'
+        'index',
+        'complete',
+        'send'
     );
 
     /**
      * An array containing the recipients basic information
      *
+     * @config
      * @var array
      */
-    public static $receiverInfo = array(
+    private static $receiver_info = array(
```

Comment:
This change makes the receiver info configurable if required, didn't see much benefit in leaving it as a public static